common/src/main/java/com/microsoft/identity/common/internal/controllers/BrokerOperationExecutor.java

```diff
@@ -69,30 +71,34 @@
         /**
          * Gets a BrokerOperationBundle bundle to pass to each IpcStrategies.
          */
-        @NonNull BrokerOperationBundle getBundle();
+        @NonNull
+        BrokerOperationBundle getBundle();
 
         /**
          * Extracts the result object from a bundle returned by an IpcStrategy.
          * If the broker returns an error, this will throw an exception.
          */
-        @NonNull T extractResultBundle(final @Nullable Bundle resultBundle) throws BaseException;
+        @NonNull
+        T extractResultBundle(final @Nullable Bundle resultBundle) throws BaseException;
 
         /**
          * Returns method name (for logging/telemetry purpose).
          */
-        @NonNull String getMethodName();
+        @NonNull
+        String getMethodName();
 
         /**
          * ID of the telemetry API event associated to this strategy task.
          * If this value returns null, no telemetry event will be emitted.
          */
-        @Nullable String getTelemetryApiId();
+        @Nullable
+        String getTelemetryApiId();
 
         /**
          * A method that will be invoked before the success event is emitted.
          * If the calling operation wants to put any value in the success event, put it here.
          */
-        void putValueInSuccessEvent(final @NonNull ApiEndEvent event,final @NonNull T result);
+        void putValueInSuccessEvent(final @NonNull ApiEndEvent event, final @NonNull T result);
```

Comment:
```suggestion
        void putValueInSuccessEvent(@NonNull final ApiEndEvent event, @NonNull final T result);
```

(annots before reserved words - here and everywhere)